demos/python_demos/audio_classification_demo/README.md

```diff
@@ -0,0 +1,60 @@
+# Audio Classification Python\* Demo
+
+Demo application for audio classification algorithm.
+
+## How It Works
+
+Upon the start-up the demo application reads command line parameters and loads a network to Inference engine. It uses only audiofiles in `wav` format. Audio could be converted to different samplerate using `-sr/--samplerate` option, if necessary for the model. After reading the audio, it is sliced into clips to fit model input (clips are allowed to overlap with `-ol/--overlap` option) and each clip is processed separately with its own prediction. 
+
+## Running
+
+Run the application with the `-h` option to see the usage message:
+```
+python3 audio_classification_demo.py -h
+```
+The command yields the following usage message:
+```
+usage: audio_classification_demo.py [-h] -i INPUT -m MODEL [-l CPU_EXTENSION]
+                                    [-d DEVICE] [--labels LABELS]
+                                    [-fr SAMPLERATE] [-ol OVERLAP]
+
+Options:
+  -h, --help            Show this help message and exit.
+  -i INPUT, --input INPUT
+                        Required. Input to process
+  -m MODEL, --model MODEL
+                        Required. Path to an .xml file with a trained model.
+  -l CPU_EXTENSION, --cpu_extension CPU_EXTENSION
+                        Optional. Required for CPU custom layers. Absolute
+                        path to a shared library with the kernels
+                        implementations.
+  -d DEVICE, --device DEVICE
+                        Optional. Specify the target device to infer on; CPU,
+                        GPU, FPGA, HDDL or MYRIAD is acceptable. The sample
+                        will look for a suitable plugin for device specified.
+                        Default value is CPU
+  --labels LABELS       Optional. Labels mapping file
```

Comment:
What is the format of this mapping file?
We have a JSON file that has the label mapping in the internal repository. Let me know if you would like to use it.

Reply:
@sathishkuttan yes, I think mapping file could be useful to distribute with OMZ

Reply:
@sathishkuttan it shoud be simple txt file each line of it represent class name. I could add support for JSON files, but I tried to make demo more universal w/o model binding.
@vladimir-dudnik IMHO such file should be distributed with model

Reply:
@eizamaliev ok, let's put labels file with model